### ISSUE TITLE
Rule style_hover_persistent

### ISSUE THREAD

**Issue by aliunwala** [IBMa/equal-access#661]
Determine which rules we can create for detecting hover issues. See Box for background:
https://ibm.ent.box.com/file/521562780202

——————
- [x] research previous notes, possible ACT rules
- [x] general rule description (SME & Dev)
- [x] what parts of the rule we want to implement (scope)
- [x] failure cases (from which I will implement code that triggers on these cases and formal test cases) (would contain Violations and / or Needs Review and / or Recommendations)
- [x] rule name
- [x] rule set and level

Original internal issue:  https://github.ibm.com/IBMa/accessibility-web-engine/issues/197

one of the goals or outcomes of this initial analysis and squad meeting is to determine what should be the priorities of any possible rule development actions now that we are getting close to the end of the release.
